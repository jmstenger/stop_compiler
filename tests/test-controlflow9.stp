/*
 * test-controlflow9.stp
 * =====================
 */

def main = ():Int {
	var i:Int;
	var j:Int;
	for (i = 0; i <5; i = i +1) {
		printf("%d", i);

		for (j=i; j>0 ; j=j-1){
			printf("%d",j);
		}
	}

	return 0;
}
