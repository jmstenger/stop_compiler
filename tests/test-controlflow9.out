011221332144321
